sdk/ai/azure-ai-projects/FunctionTools.md

```diff
@@ -0,0 +1,92 @@
+# Function Tool Specifications
+Function tools are the utility allowing developers to provide functions within their code and invoke during streaming or running.   
```

Comment:
I would use FunctionTool here as that is the API and which customer need to know

Reply:
Resolved